walkthroughs/howto-cross-account/primary-account/app.yaml

```diff
@@ -228,15 +199,14 @@ Resources:
             awslogs-stream-prefix: 'gateway-envoy'
         Environment:
         - Name: 'APPMESH_VIRTUAL_NODE_NAME'
```

Comment:
We should update this to `APPMESH_RESOURCE_ARN` as specified in: https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html